um_canonicalize/tags.yaml

```diff
@@ -56,17 +56,22 @@ categories:
     - BE2P
     - BE3S
     - BE3P
+    - ARGAC3S
```

Comment:
Nearly all the new tags, and in particular those that are not combinations of others, should be documented somewhere---here would do for now. What is this tag?

Reply:
The accusative argument is 3rd singular. (It's also an example that JCS gave.) It's used only in Navajo.